proof/refine/ARM/Ipc_R.thy

```diff
@@ -4005,7 +4452,8 @@ crunches replyPush
   and tcb_at'[wp]: "tcb_at' t"
   and cte_wp_at'[wp]: "cte_wp_at' P p"
   and ctes_of[wp]: "\<lambda>s. P (ctes_of s)"
-  (wp: crunch_wps hoare_vcg_all_lift valid_irq_node_lift simp: crunch_simps valid_mdb'_def)
+  (wp: crunch_wps hoare_vcg_all_lift valid_irq_node_lift simp: crunch_simps valid_mdb'_def
+       bindScReply_def)
```

Comment:
Unfolding `bindScReply_def` in a crunch seems strange to me, especially without also ignoring it. What happens if you don't have it?

Reply:
Good point, I'll check it out!